src/ecrc-frontend/src/components/base/termsOfUse/TermsOfUse.js

```diff
@@ -5,14 +5,34 @@ import { Button } from "../button/Button";
 
 import "./TermsOfUse.css";
 
-export default function TermsOfUse({ onClick }) {
+export default function TermsOfUse({
+  onContinueClick,
+  checkFirstBox,
+  checkSecondBox,
+  termOfUseOnScroll,
+  continueBtnEnabled
+}) {
   const button = {
     label: "Continue",
-    buttonStyle: "btn btn-primary",
-    buttonSize: "btn btn-sm",
+    buttonStyle: "btn ecrc_go_btn",
+    buttonSize: "btn ",
+    type: "submit",
+    disabled: !continueBtnEnabled
+  };
+
+  const cancelButton = {
+    label: "Cancel and Exit",
+    buttonStyle: "btn ecrc_accessary_btn",
+    buttonSize: "btn",
     type: "submit"
   };
 
+  const onCancelClicked = () => {
+    window.location.replace(
```

Comment:
why is this still using window.location.replace if you added a route?

Reply:
code updated